public/jmap/startup-options.d.ts

```diff
@@ -428,4 +428,31 @@ declare interface JCoreOptions {
    * In JMap NG App, no basemap panel will be displayed in the left panel
    */
   disableBasemaps?: boolean
+
+  /**
+   * A basemap id.
+   *
+   * If provided, will use the corresponding basemap as the default basemap.
+   *
+   * If the id is unknown, will do nothing. To get all basemaps ids in the project, use [[JMap.Map.Basemap.getAllIds()]]
+   *
+   * Using the id "none" will disable the default basemap.
```

Comment:
Je dirais pas "disable the default basemap", ça sous-entend qu'on ne pourra pas l'utiliser.  peut-être "will use no basemap at startup"? ou quelque chose du genre